src/manual/peer2peer/js/main.js

```diff
@@ -411,7 +411,12 @@ function screenCaptureExtensionHandler_() {
     // user chose a stream
     if (event.data.type && (event.data.type === 'SS_DIALOG_SUCCESS')) {
       var constraints = {
-        audio: false,
+        audio: {
+          mandatory: {
+            chromeMediaSource: 'desktop',
```

Comment:
Sorry was not clear, I think it would make it more concise if we had a version check here to ;)